apps/labs/posts/tdk-partners-with-quansight-labs.md

```diff
@@ -0,0 +1,134 @@
+category: [IDEs]
+featuredImage:
+  src: /posts/tdk-partners-with-quansight-labs/blog_feature_var1.svg
+  alt: 'An illustration of a brown and a dark brown hand coming towards each other to pass a business card with the logo of Quansight Labs.'
+hero:
+  imageSrc: /posts/tdk-partners-with-quansight-labs/blog_hero_org.svg
+  imageAlt: 'An illustration of a white hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+---
+
+TDK-Micronas is sponsoring Spyder development efforts through Quansight Labs.
+This will enable the development of some features that have been requested by
+our users, as well as new features that will help TDK develop custom Spyder
+plugins in order to complement their Automatic Test Equipment (ATE’s) in the
+development of their Application Specific Integrated Circuits (ASIC’s).
+
+At this point it may be useful to clarify the relationship the role of
+Quansight Labs in Spyder's development and the relationship with TDK. To quote
+Ralf Gommers (director of Quansight Labs):
+
+>"We're an R&D lab for open source development of core technologies around data
+science and scientific computing in Python. And focused on growing communities
+around those technologies. That's how I see it for Spyder as well: Quansight
+Labs enables developers to be employed to work on Spyder, and helps with
+connecting them to developers of other projects in similar situations. Labs
+should be an enabler to let the Spyder project, its community and individual
+developers grow. And Labs provides mechanisms to attract and coordinate
+funding. Of course the project is still independent. If there are other
+funding sources, e.g. donations from individuals to Spyder via OpenCollective,
+all the better."
+
+## Multiple Projects aka Workspaces
+
+In its current state Spyder can only handle one active project at a time.
+Although in the past we had basic support for workspaces, it was never a fully
+functional feature, so to ease development and simplify the user experience,
+we decided to remove it in the 3.x series.
+
+For TDK-Micronas, as well as for a large part of the Spyder user base, the use
+of multiple simultaneous projects is a common pattern, so this enhancement
+will:
+
+* Move from a single to a multiple active projects concept
+* Change the name `Project Explorer` to `WorkSpace Explorer`.
+* Copying from one project to another is done in the 'Explorer' by means of
+  context menus.
+
+It is worth mentioning that projects are considered an “advanced feature”,
+so it will not be forced on users that just want to get some work done.
+Projects usage is, and will remain, an optional feature.
+
+## Project Types
+
+Since the revamp of Spyder Projects with the 3.x series, the new dialog hinted
+at the possibility of creating different types of projects. Currently Spyder
+provides an "empty project" project type.
+
+![Project creation](/posts/tdk-partners-with-quansight-labs/spyder-project-creation.png)
```

Comment:
@MarsBarLee this image isn't in the preview. It shows the text rather than the image.

Reply:
All three body images now show after rebuilding. Looks like it needed an another try, like how a previous build problem went away after an adding and removing a small change.